client/src/components/ViewModeToggle/ViewModeToggle.scss

```diff
@@ -0,0 +1,78 @@
+.viewModeToggle {
+  margin-left: auto;
+
+  #splitModeButton {
+    &:disabled {
+      pointer-events: all;
+    }
+  }
+}
+
+.dropdown {
+  .dropdown-menu.show {
+    min-width: 12rem;
+
+    .btn {
+      margin-left: 0;
+    }
+
+    .viewMode-selected.btn {
+      &,
+      &:hover,
+      &:before,
+      &.highlighted,
+      &:focus {
+        background: $dropdown-link-active-bg;
+        color: $component-active-color;
+        border: 0;
+      }
+    }
+  }
+}
+
+.disabled-tooltip {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  height: 37px;
+  background: rgba(255, 255, 255, 0);
```

Comment:
I think you could probably use some variables for this, and for the `#555` lower. See `$gray-darker` etc in variables.scss and maybe use `darken($gray-dark, 10%)` if you need to